core/src/scene/styleContext.cpp

```diff
@@ -233,7 +233,6 @@ bool StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
             case StyleParamKey::color:
             case StyleParamKey::outline_color:
             case StyleParamKey::font_fill:
-            case StyleParamKey::font_stroke:
```

Comment:
Hmm I believe that documentation is wrong. Should only accept:

`stroke: { color: white, width: 5px }`

cc @meetar

On Fri, Sep 25, 2015 at 2:43 PM, Varun <notifications@github.com> wrote:

> In core/src/scene/styleContext.cpp
> <https://github.com/tangrams/tangram-es/pull/303#discussion_r40462560>:
>
> > @@ -233,7 +233,6 @@ bool StyleContext::parseStyleResult(StyleParamKey _key, StyleParam::Value& _val)
> >              case StyleParamKey::color:
> >              case StyleParamKey::outline_color:
> >              case StyleParamKey::font_fill:
> > -            case StyleParamKey::font_stroke:
>
> Documentation
> <https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#stroke>,
> says stroke can be a scalar value, which will get mapped to the color of
> the stroke. But I am not sure how useful this is!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram-es/pull/303/files#r40462560>.
>